src/ranged.cpp

```diff
@@ -757,16 +777,14 @@ static int print_steadiness( const catacurses::window &w, int line_number, doubl
 static double confidence_estimate( int range, double target_size,
                                    const dispersion_sources &dispersion )
 {
-    // This is a rough estimate of accuracy based on a linear distribution across min and max
-    // dispersion.  It is highly inaccurate probability-wise, but this is intentional, the player
-    // is not doing Gaussian integration in their head while aiming.  The result gives the player
-    // correct relative measures of chance to hit, and corresponds with the actual distribution at
-    // min, max, and mean.
-    if( range == 0 ) {
+    // 5/1/2019 - Updated to reflect normal distribution changes.
+    if( range == 0 ) { // does this ever happen?
         return 2 * target_size;
     }
-    const double max_lateral_offset = iso_tangent( range, dispersion.max() );
-    return 1 / ( max_lateral_offset / target_size );
+    // dispersion is the stdev of the distribution. Convert to units of tiles:
+    const double stdev_tiles = iso_tangent( range, dispersion.stdev() );
+    // CDF from https://en.wikipedia.org/wiki/Half-normal_distribution
+    return erf( target_size * 0.5 / ( stdev_tiles * sqrt(2) ) );
```

Comment:
Re-read and restore the previous comment and estimate.  This is not precise BY DESIGN.
If it needs to be recentered around 0 or whatever, that's ok, but you're injecting way too much smarts into this feedback mechanism.